docs/src/main/sphinx/connector/hive-s3.rst

```diff
@@ -310,14 +310,16 @@ Property name                         Description
 
 ``hive.s3.connect-timeout``           TCP connect timeout.                                        ``5 seconds``
 
+``hive.s3.connect-ttl``               TCP connect ttl, which affects connection reusage.          By default, connections do not expire.
```

Comment:
Uppercase "ttl" in Description column. "By default" is redundant because it's in Default column.